docs/component.md

```diff
@@ -79,15 +79,19 @@ a [can-component/can-template] that is used to render the search results:
 
 @signature `new Component([options])`
 
-Create an instance of a component without rendering it in a template.
+Create an instance of a component without rendering it in a template. This is
+useful when you:
+
+- have complex logic for switching between different components (e.g. routing)
+- want to create components without adding them to the page (e.g. testing)
```

Comment:
Check out the changes in this section on the 5.0 site: https://canjs.github.io/next/doc/can-component.html#newComponent__options__